log/handlers.ts

```diff
@@ -50,7 +51,46 @@ export class BaseHandler {
   async destroy() {}
 }
 
+interface ConsoleHandlerOptions extends HandlerOptions {
+  colors: boolean;
+}
+
 export class ConsoleHandler extends BaseHandler {
+  colors = true;
+
+  constructor(levelName: string, options?: ConsoleHandlerOptions) {
+    super(levelName);
+
+    if (options && !options.colors) {
```

Comment:
This conditional will disable colors if they are not explicitly set to `true` in options. I think proper way would be:
`options && options.colors === false`